### PR TITLE
Bot API 5.1

### PR DESCRIPTION
**Pull request by Olegt0rr**

### CODE REVIEW

**Review by JrooTJunior — APPROVED** [review]

**Review by JrooTJunior — APPROVED** [review]

File: aiogram/bot/bot.py [hunk 1]
```diff
@@ -1789,6 +1814,100 @@ async def export_chat_invite_link(self, chat_id: typing.Union[base.Integer, base
         result = await self.request(api.Methods.EXPORT_CHAT_INVITE_LINK, payload)
         return result
 
+    async def create_chat_invite_link(self,
+                                      chat_id: typing.Union[base.Integer, base.String],
+                                      expire_date: typing.Union[base.Integer, datetime.datetime,
+                                                                datetime.timedelta, None],
+                                      member_limit: typing.Optional[base.Integer],
```
Comment (evgfilim1) [inline aiogram/bot/bot.py:1821]: I guess `= None`s are required here

File: aiogram/bot/bot.py [hunk 2]
```diff
@@ -1789,6 +1814,100 @@ async def export_chat_invite_link(self, chat_id: typing.Union[base.Integer, base
...
+
+        result = await self.request(api.Methods.CREATE_CHAT_INVITE_LINK, payload)
+        return result
+
+    async def edit_chat_invite_link(self,
+                                    chat_id: typing.Union[base.Integer, base.String],
+                                    invite_link: base.String,
+                                    expire_date: typing.Union[base.Integer, datetime.datetime,
+                                                              datetime.timedelta, None],
+                                    member_limit: typing.Optional[base.Integer],
```
Comment (evgfilim1) [inline aiogram/bot/bot.py:1857]: Same here

File: aiogram/types/chat.py [hunk 3]
```diff
@@ -185,30 +186,47 @@ async def set_description(self, description: base.String) -> base.Boolean:
...
-        :param until_date: Date when the user will be unbanned, unix time.
-        :type until_date: :obj:`typing.Optional[base.Integer]`
-        :return: Returns True on success.
+
+        :param until_date: Date when the user will be unbanned. If user is banned
+            for more than 366 days or less than 30 seconds from the current time they
+            are considered to be banned forever. Applied for supergroups and channels
+            only.
+        :type until_date: :obj:`typing.Union[base.Integer, datetime.datetime,
+            datetime.timedelta, None`
```
Comment (evgfilim1) [inline aiogram/types/chat.py:213]:
```suggestion
            datetime.timedelta, None]`
```

File: aiogram/types/chat.py [hunk 4]
```diff
@@ -554,6 +572,41 @@ async def export_invite_link(self) -> base.String:
 
         return self.invite_link
 
+    async def create_invite_link(self,
+                                 expire_date: typing.Union[base.Integer, datetime.datetime,
+                                                           datetime.timedelta, None],
+                                 member_limit: typing.Optional[base.Integer],
```
Comment (evgfilim1) [inline aiogram/types/chat.py:578]: Same for `None` here

File: aiogram/types/chat.py [hunk 5]
```diff
@@ -554,6 +572,41 @@ async def export_invite_link(self) -> base.String:
...
+            chat_id=self.id,
+            expire_date=expire_date,
+            member_limit=member_limit,
+        )
+
+    async def edit_invite_link(self,
+                               invite_link: base.String,
+                               expire_date: typing.Union[base.Integer, datetime.datetime,
+                                                         datetime.timedelta, None],
+                               member_limit: typing.Optional[base.Integer],
```
Comment (evgfilim1) [inline aiogram/types/chat.py:591]: … and here

**Comment by evgfilim1** [comment]
@JrooTJunior I've noticed some small issues, we need to fix them

**Comment by JrooTJunior** [comment]
@evgfilim1, ty
Fixed in https://github.com/aiogram/aiogram/commit/76955bf8f1b0e67d6ada302ffd7c3ffe4d9c7a18

**Pull request merged**